example-zowe.yaml

```diff
@@ -317,7 +278,6 @@ zowe:
   cookieIdentifier: "1"
 
   # >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-  # **COMMONLY_CUSTOMIZED**
   # You can list your external domains on how you want to access Zowe.
   # This should be the domain list you would like to put into your web browser's
```

Comment:
Would it be possible to remove all single apostrophes from comments? If you are editing this under ISPF (I am not doing this, but the customers do), then `hi c` is useful - it highlights the line comments as preprocessor statements and make it readable. But the single apostrophes are braking this.